src/components/upload/FileDropZone.js

```diff
@@ -1,27 +1,28 @@
+import { color } from "@mui/system";
 import DropZone from "react-dropzone";
-import styled from "styled-components";
+import {styled} from '@mui/system';
 
-const DropZoneRoot = styled.div`
-  width: 100%;
-  height: 200px;
-  border: 4px dashed #AFAFAF;
-  border-radius: 16px;
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-  align-items: center;
-  cursor: pointer;
-  padding: 18px;
-  
-  :hover {
-    background-color: #F8F8F8;
-    border: 4px solid #AFAFAF;
-  }
-`;
+const DropZoneRoot = styled('div')({
+     width : '100%',
+     height: 200,
+     border: 4 ,
+     borderStyle : 'dashed' ,
+     borderColor : '#AFAFAF',
+     display : "flex",
+     flexDirection : "column",
+     justifyContent : "space-between",
+     alignItems : "center",
+     cursor : "pointer", 
+     '&:hover' : {
+        background: '#FFFFFF',
+        border: 4 ,
+        borderColor : '#AFAFAF'
+     }
 
-const FileNameDisplay = styled.div`
-  color: #888888;
-`;
+  });
+  const FileNameDisplay = styled('div')({
+     color : '#888888'
+  });
```

Comment:
The indentation here and in all lines from `l.6` on is wrong. Indentation should be multiples of 4 spaces (in this case 0 or 4, respectively)

Reply:
Indentation is still wrong. And I was mistaken: 2 spaces should be the indentation here and not four.